domains/kalmed.localplayer.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Gamer Profile Website",
+    "domain": "localplayer.dev",
+    "subdomain": "kalmed",
+  
+    "owner": {
+        "repo": "https://github.com/k4lm3d/k4lm3d.github.io",
+        "email": "k4lm3d@duck.com"
+    },
+
+    "record": {
+        "A": ["185.199.108.153", "185.199.109.153", "185.199.110.153", "185.199.111.153"],
+        "CNAME": "k4lm3d.github.io"
+    },
```

Comment:
Cannot have CNAME in conjunction with other records and vice versa per DNS rules.